src/js/base/ScrollList.tsx

```diff
@@ -31,19 +38,29 @@ type ScrollListProps = {
  * @param isLoading - Whether the first page is being fetched
  * @param items - The list of items
  * @param renderRow - A function which accepts an item and returns a react element
+ * @param elementId - The element identifier to scroll within
  * @returns An infinitely scrolling list of items
  */
 
-export const ScrollList = ({ fetchNextPage, isFetchingNextPage, isLoading, items, renderRow }: ScrollListProps) => {
+export const ScrollList = ({
+    fetchNextPage,
+    isFetchingNextPage,
+    isLoading,
+    items,
+    renderRow,
+    elementId,
+}: ScrollListProps) => {
     useEffect(() => {
+        const scrollListElement = elementId ? document.getElementById(elementId) : window;
```

Comment:
This works but I don't think having to pass the outer component id is very intuitive as an API. Can you apply the overflow property on the scrollist and using the scrollist itself as the element? Might even make the height determination calculation easier.